draft-ietf-tls-dtls-connection-id.md

```diff
@@ -249,45 +249,63 @@ as follows:
 # Examples
 
 {{dtls-example2}} shows an example exchange where a connection id is
-used uni-directionally from the client to the server.
+used uni-directionally from the client to the server. To indicate that 
+a connection_id has zero length we use the term 'connection_id=empty'.
 
 ~~~~
 Client                                             Server
 ------                                             ------
 
-ClientHello
-(connection_id=empty)
-                            -------->
+ClientHello                 -------->
+(connection_id=empty)       
 
 
                             <--------      HelloVerifyRequest
                                                      (cookie)
 
-ClientHello                 -------->
+ClientHello                 --------> 
 (connection_id=empty)
-  +cookie
+(cookie)                   
 
-                            <--------             ServerHello
+                                                  ServerHello
                                           (connection_id=100)
                                                   Certificate
                                             ServerKeyExchange
                                            CertificateRequest
-                                              ServerHelloDone
+                            <--------         ServerHelloDone
 
-Certificate                 -------->
+Certificate                 
 ClientKeyExchange
 CertificateVerify
 [ChangeCipherSpec]
-Finished
-(cid=100)
-                            <--------      [ChangeCipherSpec]
-                                                     Finished
-
-Application Data           ========>
-(cid=100)
-                           <========         Application Data
+Finished*                   -------->
+<cid=100>                   
+
+                                           [ChangeCipherSpec]
+                            <--------                Finished
+
+
+Application Data**          ========>
+<cid=100>
+
+                            <========        Application Data
```

Comment:
Please add a comment to clarify issue #22.
Should the new record layer be used (https://github.com/tlswg/dtls-conn-id/blob/master/draft-ietf-tls-dtls-connection-id.md#record-layer-extensions), even if the cid is empty? Or should the RFC6347 record layer be used?

Reply:
The empty cid is done in PR #35.